src/bundle/Form/TranslationAddDataTransformer.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\AutomatedTranslation\Form;
+
+use Ibexa\AdminUi\Form\Data\Content\Translation\TranslationAddData as BaseTranslationAddData;
+use Ibexa\Bundle\AutomatedTranslation\Form\Data\TranslationAddData;
+use Symfony\Component\Form\DataTransformerInterface;
+
+class TranslationAddDataTransformer implements DataTransformerInterface
+{
+    /**
+     * @param \Ibexa\AdminUi\Form\Data\Content\Translation\TranslationAddData $value
+     *
+     * @return \Ibexa\Bundle\AutomatedTranslation\Form\Data\TranslationAddData
+     */
+    public function transform($value)
```

Comment:
```suggestion
    public function transform($value): TranslationAddData
```

but implementation make a lot of optimistic assumptions about \`$value\`